@here/harp-webpack-config/index.js

```diff
@@ -0,0 +1,98 @@
+const path = require("path");
+const merge = require("webpack-merge");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+
+function addHarpWebpackConfig(config, mainPath, decoderPath, htmlPath) {
+    if (Array.isArray(config) || typeof config === "function") {
+        throw new Error(
+            "config passed to addHarpWebpackConfig must be an object, not an array or function."
+        );
+    }
+    const userConfig = config ? config : {};
+    const commonConfig = {
+        output: {
+            filename: "[name].bundle.js",
+            path: path.join(__dirname, "dist")
+        },
+        devtool: "source-map",
+        externals: [
+            {
+                three: "THREE"
```

Comment:
Actually this break pure javascript projects created with the generator.
We may need to add a parameter for project type.